templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -42,20 +42,44 @@
     "from ark.utils import data_utils, example_dataset, plot_utils\n",
```

Comment:
<p>To be consistent with the way this path is specified for the example dataset above, can we change<code>{base_dir}/segmentation/cell_table/cell_table_size_normalized.csv</code>&nbsp;to&nbsp;<code>../data/example_dataset/segmentation/cell_table/cell_table_size_normalized.csv</code>? And also get rid of the quotes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>